kokoro/gcp_ubuntu/bazel/build.sh

```diff
@@ -14,7 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Build the project with bazel using Kokoro.
+# For use within a IREE bazel docker image on a Kokoro VM.
+# Log some infomration about the environment, initalize the submodules and then
```

Comment:
Typo :-D
```suggestion
# Log some information about the environment, initialize the submodules and then
```

Reply:
My editor's spell checker has just decided not to work, revealing that I can't spell 😅